webrisk/snippets/compute_threatlist_diff.py

```diff
@@ -22,7 +23,7 @@ def compute_threatlist_diff(
     max_diff_entries: int,
     max_database_entries: int,
     compression_type: webrisk_v1.CompressionType,
-) -> None:
+) -> ComputeThreatListDiffResponse:
     """Gets the most recent threat list diffs. These diffs should be applied to a local database of
```

Comment:
It's preferable to have a single line beginning the docstring, with more details below.

Reply:
Done, thanks!